release_management/create_artifacts.py

```diff
@@ -95,8 +96,8 @@ def _stratisd_artifacts(namespace):
             "Need script to run at top-level of package, in same directory as Cargo.toml"
         )
 
-    output_abs_path = os.path.abspath(namespace.output_dir)
-    os.makedirs(output_abs_path, exist_ok=True)
+    output_path = namespace.output_dir
```

Comment:
In the parser, where we define the ```output_dir``` argument using ```add_argument``` add an additional keyword argument, thus ```type=os.path.abspath```. This will ensure that ```output_path``` is always an absolute path when the parser namespace is passed to any other functions.